### ISSUE TITLE
Additional output-folders are ignored

### ISSUE THREAD

**Issue by rudibarani** [jlesage/docker-handbrake#191]
I have mounted a second watch-folder `/watch2` and `/output2` folder to my docker image. They are accessible for rw.
I have added `AUTOMATED_CONVERSION_OUTPUT_DIR_2` = `/output2` to the environment variables.

If I add a file to the `/watch2` folder, the file is processed and saved to the regular `/output` folder instead of `/output2`.

Am I doing something wrong here?

**Comment by grumpyspud** [comment]
Not sure if you resolved this or not, but I was having a similar issue while trying to have two different watch folders that were mounted from two different network shares output to two different output folders also mounted to different shares. After checking the logs for the container, I found that it was saying the output folder specified by AUTOMATED_CONVERSION_OUTPUT_DIR_2 was reporting that it didn't exist. After checking that it did exist outside the container, I realized that the path specified by that env var is relative to the container, not the system/host. This means that I needed the output shares to be under the /output path being passed to the container with the -v /output var.

I ended up making a directory called 'output' on the system and then creating two folders in it for each of my two desired output folders ('share1', 'share2'). I then mounted the shares for each output to their corresponding  'output/share' dir. Then I used the AUTOMATED_CONVERSION_OUTPUT_DIR to point to /output/share1 and AUTOMATED_CONVERSION_OUTPUT_DIR_2 to point to /output/share2 with -v /path/to/output/folder:/output:rw. This made it so that anything auto-encoded from each watch folder will go to the correct output folder and use the specified encoder preset for each. Anything that is manually added/encoded using the web interface will need to be specified to which output folder it needs to be sent to or it will end up on the host system in the folder specified in the -v /path/to/output/folder option.

Hope this helps.
